platform/rct/hal/usart_stm32/usart_device_stm32.c

```diff
@@ -66,27 +68,27 @@ static int initQueues()
     int success = 1;
 
     /* Create the queues used to hold Rx and Tx characters. */
-    xUsart0Rx = xQueueCreate(UART_QUEUE_LENGTH,
+    xUsart0Rx = xQueueCreate(UART_RX_QUEUE_LENGTH,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
-    xUsart0Tx = xQueueCreate(UART_QUEUE_LENGTH + 1,
+    xUsart0Tx = xQueueCreate(UART_TX_QUEUE_LENGTH + 1,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
     if (xUsart0Rx == NULL || xUsart0Tx == NULL) {
         success = 0;
         goto cleanup_and_return;
     }
 
-    xUsart2Rx = xQueueCreate(UART_QUEUE_LENGTH,
+    xUsart2Rx = xQueueCreate(UART_RX_QUEUE_LENGTH,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
-    xUsart2Tx = xQueueCreate(UART_QUEUE_LENGTH + 1,
+    xUsart2Tx = xQueueCreate(UART_TX_QUEUE_LENGTH + 1,
                              (unsigned portBASE_TYPE)sizeof(signed portCHAR));
     if (xUsart2Rx == NULL || xUsart2Tx == NULL) {
         success = 0;
         goto cleanup_and_return;
     }
 
-    xUsart3Rx = xQueueCreate(UART_QUEUE_LENGTH,
+    xUsart3Rx = xQueueCreate(UART_RX_QUEUE_LENGTH,
```

Comment:
Also... this is going to break quite hard with the new virtual serial code.  Oh well...